cfpb/cfgov-refresh
# DF-1806 About-Us Video

DF-1806 About Us Video

## Changes
- Updated  ' _includes/macros.html' to add youtube macro and allow share macro to accept additional classes.
- Updated  ' static/css/bureau.less' to add styles to for About-Us video.
- Updated 'src/static/js/routes/common.js' to initialize youtube.js.
- Updated 'the-bureau/index.html' to add hero block and call youtube macro.
- Added 'src/static/js/modules/youtube.js' to use Youtube api.
- Added ' static/img/youtube-play-btn.png' to provide image for playing youtube video.

## Notes
- There is a lot of markup and breakpoints.  Room for improvement and optimizations.
- Implementation of background color on share icons isn't ideal.
- Iframe insertion is causing errors denoted in #548 

## Test
- Navigate to http://localhost:7000/the-bureau/ and click 'play icon'
- Video should begin to play
- Click 'close' button and you should return to original state.

## Review
 
- @jimmynotjim  
- @anselmbradford 
- @Scotchester  
- @KimberlyMunoz 

## Screenshots
Mobile
![screen shot 2015-05-12 at 10 00 54 am](https://cloud.githubusercontent.com/assets/1696212/7588908/d6c6b31a-f88d-11e4-97d2-ab59fa355c8d.png)

Tablet
![screen shot 2015-05-12 at 10 00 45 am](https://cloud.githubusercontent.com/assets/1696212/7588906/d6c51aa0-f88d-11e4-8086-f0012c3ed70f.png)

Desktop
![screen shot 2015-05-12 at 10 00 31 am](https://cloud.githubusercontent.com/assets/1696212/7588907/d6c53daa-f88d-11e4-805e-030ef69adcf7.png)

Mobile
![screen shot 2015-05-12 at 10 04 58 am](https://cloud.githubusercontent.com/assets/1696212/7589026/696f0cd0-f88e-11e4-9c74-0cf45655e71b.png)

Tablet
![screen shot 2015-05-12 at 10 04 25 am](https://cloud.githubusercontent.com/assets/1696212/7589027/697078ea-f88e-11e4-9562-d005f5775184.png)

Desktop
![screen shot 2015-05-12 at 10 04 13 am](https://cloud.githubusercontent.com/assets/1696212/7589029/6971e040-f88e-11e4-84ea-7aa21401db19.png)
